services/userService.js

```diff
@@ -0,0 +1,22 @@
+const bcrypt = require("bcrypt");
+
+const userDao = require("../models/userDao");
+const {
+  emailValidationCheck,
+  passwordValidationCheck,
+} = require("../utils/validation-check");
+
+const signUp = async (email, password, name, phone_number) => {
+  emailValidationCheck(email);
+  passwordValidationCheck(password);
```

Comment:
```suggestion
  await emailValidationCheck(email);
  await passwordValidationCheck(password);
```
동기적으로 동작해야하는 코드들입니다!